eventing/triggers/new-trigger-filters.adoc

```diff
@@ -0,0 +1,47 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="new-trigger-filters"]
+= New trigger filters
+include::_attributes/common-attributes.adoc[]
+:context: new-trigger-filters
+
+toc::[]
+
+:FeatureName: New trigger filters feature
+include::snippets/technology-preview.adoc[]
+
+The new trigger filters feature adds a new `filters` field to triggers that aligns with the filters API field defined in the CloudEvents Subscriptions API. It allows you to specify filter expressions, where each expression evaluates to `true` or `false` for each event. 
+
+The following example shows a trigger using the new filters field:
+[source,yaml]
+----
+apiVersion: eventing.knative.dev/v1
+kind: Trigger
+metadata:
+  name: my-service-trigger
+spec:
+  broker: default
+  filters:
+    - cesql: "source LIKE '%commerce%' AND type IN ('order.created', 'order.updated', 'order.canceled')"
+  subscriber:
+    ref:
+      apiVersion: serving.knative.dev/v1
+      kind: Service
+      name: my-service
+----
+
+The `filters` field contains an array of filter expressions, each evaluating to either `true` or `false`. If any expression evaluates to `false`, the event will not be sent to the subscriber. Each filter expression uses a specific dialect that determines the type of filter and the set of allowed additional properties within the expression.
+
+To enable new trigger filters, add the `new-trigger-filters: enabled` entry in the section of the `KnativeEventing` type in the operator config map as follows:
+[source,yaml]
+----
+apiVersion: operator.knative.dev/v1beta1
+kind: KnativeEventing
+spec:
+  config:
+    features:
+      new-trigger-filters: enabled
+----
```

Comment:
The feature is already enabled 

https://github.com/openshift-knative/eventing/blob/b27a7c6c06f0dd23d6fbd2bd82196759d25c4525/config/core/configmaps/features.yaml#L41C1-L45C33

https://github.com/openshift-knative/eventing/blob/b27a7c6c06f0dd23d6fbd2bd82196759d25c4525/pkg/apis/feature/features.go#L60C3-L60C20

Reply:
```suggestion
```

Reply:
ah. than nevermind. I was think that comes w/ 1.15 - but is now on. nice!

@kaldesai so we can remove the "enable" section -it is already there and enabled

Reply:
Okay I will remove this